### PR TITLE
Create therayrayterrill.ms

### PR DESCRIPTION
**Pull request by Masterable**

### CODE REVIEW

**Comment by Masterable** [comment]
Proof of all powers --> https://imgur.com/a/F8Dl0

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		"/cast lightray"
+                "Shoots Light Ray At People!"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 12 9999 3 true",
+                "/effect ". @plr ." 5 9999 5 true",
+                "/effect ". @plr ." 11 9999 2 true",
+                "/effect ". @plr ." 21 9999 9 true",
+                "/effect ". @plr ." 10 9999 4 true"
+                "/give ". @plr ." minecraft:iron_boots 1 8:30 4:30 name:&9W&1a&9ter_Ru&1nn&9ing_&4B&eoo&4ts"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: also missing comma here

Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: missing comma here

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                "/flyspeed <number>",
+                "/fly",
+                "/transmute <number>",
+                "/thru (phases through objects)",
+                "Shoots nuclear blasts with bare hand"
+                "/invis"
+                "/vis"
+                "/intangible"
+                "/tangible"
+		"/cast lightray"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: missing comma

Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: missing comma

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                "/haste 0-5",
+                "/flyspeed <number>",
+                "/fly",
+                "/transmute <number>",
+                "/thru (phases through objects)",
+                "Shoots nuclear blasts with bare hand"
+                "/invis"
+                "/vis"
+                "/intangible"
+                "/tangible"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: missing comma

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                "/run 0-9",
+                "/haste 0-5",
+                "/flyspeed <number>",
+                "/fly",
+                "/transmute <number>",
+                "/thru (phases through objects)",
+                "Shoots nuclear blasts with bare hand"
+                "/invis"
+                "/vis"
+                "/intangible"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: missing comma

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+                "/jump 0-3",
+                "/run 0-9",
+                "/haste 0-5",
+                "/flyspeed <number>",
+                "/fly",
+                "/transmute <number>",
+                "/thru (phases through objects)",
+                "Shoots nuclear blasts with bare hand"
+                "/invis"
+                "/vis"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: missing comma

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+                "/dismountplayer",
+                "/jump 0-3",
+                "/run 0-9",
+                "/haste 0-5",
+                "/flyspeed <number>",
+                "/fly",
+                "/transmute <number>",
+                "/thru (phases through objects)",
+                "Shoots nuclear blasts with bare hand"
+                "/invis"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: missing comma

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-3",
+                "/run 0-9",
+                "/haste 0-5",
+                "/flyspeed <number>",
+                "/fly",
+                "/transmute <number>",
+                "/thru (phases through objects)",
+                "Shoots nuclear blasts with bare hand"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill]: missing comma

**Comment by malonnnn** [comment]
missing .ms file extension

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill.ms [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+                "Shoots Light Ray At People!"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 12 9999 3 true",
+                "/effect ". @plr ." 5 9999 5 true",
+                "/effect ". @plr ." 11 9999 2 true",
+                "/effect ". @plr ." 21 9999 9 true",
+                "/effect ". @plr ." 10 9999 4 true"
+                "/give ". @plr ." minecraft:iron_boots 1 8:30 4:30 name:&9W&1a&9ter_Ru&1nn&9ing_&4B&eoo&4ts", 
+                "/give ". @plr ." minecraft:gold_helmet 1 0:6 1:6 3:3 4:2 name:&cT&6h&ee &eR&ca&6y&c'&es &7Helmet",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill.ms]: remove comma pls

File: CommandHelper/LocalPackages/commands/buffs/therayrayterrill.ms [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+                "/tangible",
+		"/cast lightray",
+                "Shoots Light Ray At People!"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 12 9999 3 true",
+                "/effect ". @plr ." 5 9999 5 true",
+                "/effect ". @plr ." 11 9999 2 true",
+                "/effect ". @plr ." 21 9999 9 true",
+                "/effect ". @plr ." 10 9999 4 true"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/therayrayterrill.ms]: add comma

**Pull request merged**